loadrunner_professional/manifest.json

```diff
@@ -0,0 +1,59 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "e6b5ab52-139d-4dde-a4ad-94fedeac7f29",
+  "app_id": "loadrunner-professional",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Send your LoadRunner Professional metrics to DataDog.",
```

Comment:
I would suggest to make this slightly more specific and mention the types of data being sent in